subsys/net/openthread/rpc/common/ot_rpc_common.c

```diff
@@ -324,58 +254,34 @@ bool ot_rpc_encode_service_config(struct nrf_rpc_cbor_ctx *ctx, const otServiceC
 	return true;
 }
 
-bool ot_rpc_decode_service_config(struct nrf_rpc_cbor_ctx *ctx, otServiceConfig *config)
+void ot_rpc_decode_service_config(struct nrf_rpc_cbor_ctx *ctx, otServiceConfig *config)
```

Comment:
encode_service_config hasn't bee updated.

Reply:
Yep, it was done intentionally. I made only changes those are required for clients. The second part will be changes for servers. Since the function is used only in servers, it will be changed in the second part.

Reply:
ok, gotcha